### DISCUSSION TITLE
Collection of general ideas to inculde in the project.

### DISCUSSION THREAD

**Discussion by Abdal-EL** [Daniel-Dfg/PlagiaSight#35] [Ideas]
### I invite every person working on the project to suggest some ideas modifications to include in the project

**Comment by Daniel-Dfg** [comment]
I'd say that a top priority is Multiple file types support (.md, .pdf, .docx...).

However, I must say that we already got a lot of things to do : 
- Making the UI snappier and clearer for  (I'm in charge of it)
- Optimizations via threading and JITC (_Just in time compiling_) (I'm in charge of it, it will be the object of my next commit)
- Exception handling (showing the user a specific window when there's an error, providing an option to reset stuff...)
